Задание 4

Manasonuke opened the pull request.

Manasonuke committed:
Add files via upload

BalakinaEP commented on Project4.cpp:
```diff
@@ -0,0 +1,59 @@
+
+std::ostream& operator<<(std::ostream& os, const point& p) {
+    return os << p.x << ' ' << p.y << ' ' << p.s;
+}
```
class
все данные private

Manasonuke committed:
Update Project4.cpp

Класс Point теперь с закрытыми данными (private).
Операторы >> и << объявлены друзьями класса, чтобы иметь доступ к приватным полям. Оператор < реализован как метод класса.
